lib/mocha.js

```diff
@@ -205,6 +207,11 @@ function Mocha(options) {
    */
   this.isWorker = Boolean(options.isWorker);
 
+  this.globalSetup(options.globalSetup)
```

Comment:
the functions are in `globalSetup`/`globalTeardown`, and `enableGlobalSetup`/`enableGlobalTeardown` is consumed by the watcher.